src/strings.rs

```diff
@@ -84,6 +85,11 @@ pub(crate) fn message_content(message: &Message) -> String {
         MessagePhoto(data) => message_photo(&data.caption.text),
         MessageExpiredPhoto => gettext("Photo has expired"),
         MessageSticker(data) => message_sticker(&data.sticker.emoji),
+        MessageScreenshotTaken => gettext_f(
+            "{sender} took a screenshot!",
+            &[("sender", &message_sender(sender, true))],
+        ),
+        MessageGameScore(data) => message_game_score(data, chat, sender),
```

Comment:
Can you change this to be in the same order as in https://docs.rs/tdlib/latest/tdlib/enums/enum.MessageContent.html, please? This way it's easier to compare the two lists and check for missing variants :)